src/main/java/frc/robot/RobotContainer.java

```diff
@@ -197,38 +190,47 @@ private void configureButtonBindings() {
     // OI.getPOVButton(OI.Driver.DPAD_UP)
     //     .and(OI.getButton(OI.Driver.RBumper).negate())
     //     .whileTrue(intake.intakeToBirdhouse());
-    OI.getPOVButton(OI.Driver.DPAD_UP).whileTrue(coralScorer.scoreClockWise());
-    // OI.getPOVButton(OI.Driver.DPAD_DOWN)
-    //     .and(OI.getButton(OI.Driver.RBumper).negate())
-    //     .whileTrue(intake.ejectFromBirdhouse());
-    // intake.setDefaultCommand(intake.retractPivotCommand());
+    OI.getPOVButton(OI.Driver.DPAD_UP).whileTrue(coralScorer.scoreCommand());
+    // OI.getButton(OI.Driver.RBumper).whileTrue(intake.floorIntake());
+    OI.getButton(OI.Driver.RBumper).whileTrue(intake.floorIntake());
+    new Trigger(() -> sensors.getSensorState() != CoralEnum.NO_CORAL)
```

Comment:
We may have to refine this Trigger condition, because I think it may trigger again when attempting to move the Coral into the Scorer. Possibly just adding `&& !intake.atSetpoint(kPivotRetractAngle` is all that is needed, since if we are at the retract angle the danger of crashing a coral into the elevator has passed.